### PR TITLE
use unlisted review url if any emails sent about unlisted reviews

### LINKED ISSUE #4971: Wrong link for Superheroes in activity mail
Steps to reproduce:
1. Submit an unlisted add-on i.e https://addons-dev.allizom.org/en-US/developers/addon/test-activity-dd-on/edit
2. Click "Unlisted Review Page" link from Dev Hub for the submitted add-on
3. Leave a "request-super review"
4. Check the activity mail from https://addons-dev.allizom.org/en-US/admin/mail

Expected results:
There are no issues in generated e-mail.

Actual results:
Listed Review link page is listed in the mail for a super=request review made from an unlisted review page, for an only-unlisted add-on versions.

Notes/Issues:
- reproducing also, for add-on with mixed versions in -stage.
Verified on AMO-dev FF52(Win 7). 
Screenshot for this issue: 

#3726

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #4971

### CODE REVIEW

File: src/olympia/editors/helpers.py [hunk 1]
```diff
@@ -508,13 +508,17 @@ def get_context_data(self):
                 addon = Addon.unfiltered.get(pk=self.addon.pk)
         else:
             addon = self.addon
+        review_url_kw = (
+            {'channel': 'unlisted'} if self.version and
+            self.version.channel == amo.RELEASE_CHANNEL_UNLISTED else {})
+        review_url_kw['addon_id'] = self.addon.pk
```
Comment (diox) [inline src/olympia/editors/helpers.py]: Not a big fan of how this reads. How about:
```python
        review_url_kw = {
            'addon_id': self.addon.pk
        }
         if (self.version and 
                self.version.channel == amo.RELEASE_CHANNEL_UNLISTED):
            review_url_kw['channel'] = 'unlisted'
```

**Comment by diox** [comment]
Is there a test you can modify to check for this change ?

**Comment by eviljeff** [comment]
yeah, there will be - I forgot.

File: src/olympia/editors/helpers.py (hunk 1 above)
Comment (eviljeff) [inline src/olympia/editors/helpers.py]: https://github.com/mozilla/addons-server/pull/5047/commits/c9c9ca760eb254bb6f98020eb35d7c8fab1a6645 with tests this time

**Pull request merged**